source/quick-start-sinatra.txt

```diff
@@ -24,6 +24,9 @@ This guide shows you how to use {+odm+} in a new Sinatra web application,
 connect to a MongoDB cluster hosted on MongoDB Atlas, and perform
 read and write operations on the data in your cluster.
 
+To learn how to migrate an existing application to use {+odm+}, see the
```

Comment:
S: my ear says you either migrate to Mongoid or adapt the application to use Mongoid, but not a combo

Reply:
THis is meant to differentiate between starting a new app or integrating Mongoid into your existing app. I can make that more clear